pkg/node/node.go

```diff
@@ -683,14 +684,20 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 }
 
 func (b *Bee) Shutdown(ctx context.Context) error {
-	errs := new(multiError)
-
-	if b.apiCloser != nil {
-		if err := b.apiCloser.Close(); err != nil {
-			errs.add(fmt.Errorf("api: %w", err))
+	var mErr error
+
+	// tryClose is a convenient closure which decrease
+	// repetitive io.Closer tryClose procedure.
+	tryClose := func(c io.Closer, errMsg string) {
+		if c != nil {
+			if err := c.Close(); err != nil {
+				mErr = multierror.Append(mErr, fmt.Errorf("%s: %w", errMsg, err))
+			}
 		}
```

Comment:
it's really a matter of preference, but I would go with an early return if `c==nil`